src/main/java/com/softserve/kickscootertrip/controller/TripController.java

```diff
@@ -0,0 +1,30 @@
+package com.softserve.kickscootertrip.controller;
+
+
+import com.softserve.kickscootertrip.model.dto.UserDto;
+import com.softserve.kickscootertrip.service.TripService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/app")
+@RequiredArgsConstructor
+public class TripController {
+
+    private final TripService tripService;
+
+    @PostMapping("/user/start")
```

Comment:
```suggestion
    @PostMapping("/users/{usersid}/trips/{tripsid}/start")
```
change endpoint to more specific one(example above)

Reply:
👍for id placeholder use camelCase. 
I wonder whether `start` verb is needed... Does it make sense to create a trip and not start it? If not, then we can get rid of the `start` verb.

Reply:
yet another point, It seems`/trips` should be a top-level endpoint because trips aren't part of the user. If you need to get trips of the specific user - the request should be: `/trips?userId=<id>`.